File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int i=s.length()-1;
        int count=0;
        while(i>=0 && s.charAt(i)==' '){
            i--;
        }
        while(i>=0){
            if(s.charAt(i)==' '){
                break;
            }
            else{
                count++;
            }
            i--;
        }
        return count;
    }
}

File: 58-length-of-last-word/length-of-last-word.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int i=s.Length-1;
        int count=0;
        while(i>=0 && s[i]==' '){
            i--;
        }
        while(i>=0){
            if(s[i]==' '){
                break;
            }
            else{
                count++;
            }
            i--;
        }
        return count;
    }
}